ui/app/src/views/home/ImportantDashboards.tsx

```diff
@@ -59,7 +60,7 @@ export function ImportantDashboards(): ReactElement {
     <Stack>
       <Stack direction="row" alignItems="center" gap={1}>
         <ViewDashboardIcon />
-        <h2>Important Dashboards</h2>
+        <h2><Trans i18nKey="Perses_lblImportantDashboards" /></h2>
```

Comment:
Although keys might be required for longer translations, is easier to reason about the translated text using an english base, later we could generate the translation files.
```suggestion
        <h2><Trans>Important Dashboards</Trans></h2>
```

Reply:
can you please expound on what you mean by "for longer translations"

Reply:
I mean longer texts like paragraphs. In those cases it makes sense to create a shorter key, for short texts the text can be the key.